src/features/todos.ts

```diff
@@ -1,8 +1,15 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 import { Todo } from '../types/Todo';
 
 export const todosSlice = createSlice({
   name: 'todos',
   initialState: [] as Todo[],
-  reducers: {},
+  reducers: {
+    setTodos: (state, action: PayloadAction<Todo[]>) => {
+      state.push(...action.payload);
```

Comment:
Using `state.push(...action.payload)` will append the new todos to the existing state. If the intention is to replace the current state with the new todos, consider using `state.splice(0, state.length, ...action.payload)` or `return action.payload` instead.